app/controllers/EventDetailsController.js

```diff
@@ -0,0 +1,10 @@
+GameUpApp.controller('EventDetailsController', ['$scope', '$location', 'eventService', 'events', function($scope, $location, eventService, events){
+  $scope.eventObj = eventService.getEventId();
+
+  $scope.formatDate = function(date) {
```

Comment:
Agreed with Tyler - in fact, ideally we should only be using the EventController to do all the event stuff - including listing, creating, getting individual events, etc. But the way it's implemented here is probably fine for now.